src/wagtail_vector_index/backends/pgvector/migrations/0003_pgvectorembedding_embedding_output_dimensions.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.7 on 2023-12-05 11:33
```

Comment:
I am probably missing a bit of context so take this with a bit of salt: given we don't have a release with the new pgvector backend, why not keep migrations to a minimum (i.e. reduce to just the initial one?)

Reply:
True! Will just keep the initial one.